.github/workflows/test.yml

```diff
@@ -136,3 +136,14 @@ jobs:
           api-level: 28
           working-directory: ./example
           script: flutter test integration_test/webcrypto_test.dart -d android
+
+  report-coverage:
+    needs: [linux, macos, windows, ios, android]
+    if: ${{ always() }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: coverallsapp/github-action@v2
```

Comment:
We always use hashes rather than `@v2`. I think it's a security consideration, see:
https://opensource.google/documentation/reference/github/services

It's also nice because then action won't stop working because it somehow changed.

If we want to upgrade actions like this we can probably get dependabot to do it. Which we can also configure in the `.github/` folder.

Reply:
updated to the hash `643bc377ffa44ace6394b2b5d0d3950076de9f63` 🙌